services/info/crawlers.ts

```diff
@@ -2667,3 +2667,18 @@ export const _uniongarmentscokr = (
     isSoldout: !result.isSoldout,
   });
 };
+
+export const _lotuffcokr = (
+  $: CheerioStatic,
+  selector: InfoSelectors
+): InfoResult => {
+  const result = selectAll($, selector);
+  const isSoldout = $(selector.isSoldout).attr('alt') == "품절";
```

Comment:
```suggestion
  const isSoldout = $(selector.isSoldout).attr('alt') === "품절";
```

의도적으로 ==을 사용한 상황이 아니라면 ===을 사용해주세요~

Reply:
네엡!
114a6aa